### ISSUE TITLE
Add the text searched in the find history.. 

### ISSUE THREAD

**Issue by bvahedy** [AndBible/and-bible#1340]
Since searches are in the history, please add the text that was searched to the end of the word "find"  in the history.
 for long text searched please truncated, so that is does not wrap.
This way if there are multiple find in the history the user can select  the desired search.

**Comment by timbze** [comment]
Relates to #831

**Comment by bvahedy** [comment]
Yes, but #831  doesn't say how to improve it. May be that is why it hasn't
been implemented or looked at. I feel it's an important mod. Otherwise you
just see the world find multiple times in the history without knowing which
is the one you want to search again.

**Comment by timbze** [comment]
I just added the note for reference. When someone looks at #831 or wants to work on it, they should see this as well.

**Comment by bvahedy** [comment]
Ok, thank you.

**Comment by agrogers** [comment]
I have had a look at this and it seems that the 'Find' history item is added when the Find window is opened not when the Find is executed.  So there is no easy fix for this one.

I am not sure how the history interacts and I fear that forcing another history item into the list  when a find is executed may mess with the Back button operation. I have added a different issue to try and address the order #2026. I can then look at integrating it better with the search.
